packages/core-utils/src/time.js

```diff
@@ -75,10 +72,7 @@ export function formatTime(ms, options) {
  * @return {string}                   formatted text representation
  */
 export function formatSecondsAfterMidnight(seconds, timeFormat) {
-  return moment()
-    .startOf("day")
-    .seconds(seconds)
-    .format(timeFormat);
+  return format(getSeconds(startOfDay(seconds)), timeFormat);
```

Comment:
The call to `getSeconds` returns the number of seconds which is then passed as an argument as milliseconds past the epoch to `format`. I think this is what we're looking for:

```suggestion
  return format(add(startOfDay(new Date()), { seconds }), timeFormat);
```

Reply:
Good catch thank you! Have fixed in 55e76870